### PR TITLE
Update website

### LINKED ISSUE #2: EZ Build docs and copy over script

### LINKED ISSUE #7: Write Spider to see dead files in repo
Write a spider to look through links to find dead files in this repo.

### LINKED ISSUE #15: Tensorboard tutorial on website is not up to date.
In particular, the version on the website does not include the important pull request found here:

https://github.com/deepchem/deepchem/pull/1261

This will make it hard for users to use Tensorboard in the notebook. Fix should be as simple as just updating the notebook on the website to match that found in the deepchem github repo.

Thanks.

### LINKED ISSUE #24: Stop deploying docs
When I searched by "deepchem docs", I hitted old docs (deepchem.io/docs).

スクリーンショット 2020-06-14 16 55 46

I think it is not good for many users, 
so we should remove docs files in this repo and stop deploying.

### PR DESCRIPTION
**Pull request by nissy-dev**
Fixed #24 
Fixed #2
Fixed #7  
Fixed #15 

This PR is related to #24 

# What  I did

- [x] Remove deepchemwebsite dir
- [x] Remove website/docs dir
- [x] Remove website/_sources dir
- [x] Remove website/_statics dir
- [x] Rename website/static to website/assets
- [x] Move all images to website/assets/img
- [x] Formet html and css
- [x] Add GitHub links in header
- [x] Fix broken links
- [ ] Move js codes to website/assets/js/main.js -> I will make the anotehr PR
- [ ] Fix mobile bugs  -> I will make the anotehr PR

### CODE REVIEW

**Comment by rbharath** [comment]
Thanks for taking on this cleanup! I think this will help simplify the website structure a good bit :)

File: website/index.html [hunk 1]
```diff
@@ -1,256 +1,192 @@
...
         <div class="row">
-            <div class="col-sm-3 col-xs-6">
-                <a href="https://pande.stanford.edu/"><img src="./static/img/logos/stanford.png"></a>
-            </div>
-            <div class="col-sm-3 col-xs-6">
-                <a href="https://www.schrodinger.com/"><img src="./static/img/logos/schrodinger.png"
-                                                            style="padding:15%"></a>
-            </div>
+          <div class="col-sm-3 col-xs-6">
+            <a href="https://www.pandelab.org/"><img src="./assets/img/stanford.png"></a>
```
Comment (nissy-dev) [inline website/index.html:107]: Pande lab link is broken. The link https://www.pandelab.org is OK?

**Comment by nissy-dev** [comment]
@rbharath 
I stopped refactoring js codes and fixing moible bugs. 
I think this PR is ready to merge, so please check it! 
https://github.com/nd-02110114/deepchem.io/tree/fix-link


I will make another PR about refactoring js codes and fixing moible bugs in the next week 🙇

**Review by rbharath — APPROVED** [review]
Ok did a pass through and this looks good. Going to merge in.

**Pull request merged**